File: app/src/main/java/com/example/administrador/starwarswiki/network/ApiaryService.java
package com.example.administrador.starwarswiki.network;

import retrofit2.Call;
import retrofit2.http.Header;
import retrofit2.http.POST;
import retrofit2.http.Path;

public interface ApiaryService {

    @POST("/favorite/{id}")
    Call<Object> postFavoriteWithHeader(@Path("id") int id, @Header("Prefer") String value);

    @POST("/favorite/{id}")
    Call<Object> postFavorite(@Path("id") int id);

}

File: app/src/main/java/com/matheusfroes/swapi/network/ApiaryService.kt
package com.matheusfroes.swapi.network

import com.matheusfroes.swapi.network.data.ApiaryResponse
import kotlinx.coroutines.experimental.Deferred
import retrofit2.Response
import retrofit2.http.POST
import retrofit2.http.Path

interface ApiaryService {

    @POST("favorite/{id}")
    fun bookmarkPerson(
            @Path("id") personId: Long
    ): Deferred<Response<ApiaryResponse>>
}